mm2src/crypto/src/global_hd_ctx.rs

```diff
@@ -74,6 +74,16 @@ impl GlobalHDAccountCtx {
     pub fn derive_secp256k1_secret(&self, derivation_path: &DerivationPath) -> MmResult<Secp256k1Secret, Bip32Error> {
         derive_secp256k1_secret(self.bip39_secp_priv_key.clone(), derivation_path)
     }
+
+    /// Derives an `ExtendedPublicKey<secp256k1::PublicKey>` from [`HDAccountCtx::bip39_secp_priv_key`]
+    /// at the given `m/purpose'/coin_type'/account'` derivation path.
+    /// Remember, XPub are for accounts.
+    pub fn derive_secp256k1_xpub(
+        &self,
+        derivation_path: &DerivationPath,
+    ) -> MmResult<ExtendedPublicKey<secp256k1::PublicKey>, Bip32Error> {
+        derive_secp256k1_xpub(self.bip39_secp_priv_key.clone(), derivation_path)
```

Comment:
Maybe call the inner one `derive_secp256k1_xpub_impl` to avoid confusion?

Reply:
[ba3a4bb](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2482/commits/ba3a4bb634429986d554ec15120850ae0f92975e)